tests/test_adjointA.py

```diff
@@ -30,3 +31,30 @@ def test_acoustic(dimensions, time_order, space_order, fix_dim):
     term2 = linalg.norm(rec.data) ** 2
     print(term1, term2, ("%12.12f") % (term1 - term2), term1 / term2)
     assert np.isclose(term1 / term2, 1.0, atol=0.001)
+
+
+@pytest.mark.parametrize('dimensions', [(60, 70), (60, 70, 80)])
+@pytest.mark.parametrize('fix_dim', [True, False])
+def test_acoustic_constant(dimensions, fix_dim):
```

Comment:
FWIW, we can further parametrize this once the model is a single switch (PR #308). We can then also build explicit test-parameter matrices to manage complexity.